ironfist/installer/map_xml.xsd

```diff
@@ -81,14 +81,24 @@
   </xs:complexType>
 
   <xs:complexType name="mapVariable_t">
+    <xs:sequence>
+      <xs:element name="array" minOccurs="0" maxOccurs="unbounded">

```

Comment:
So, "arrays" and "maps" are actually just "tables" in Lua. Should name accordingly; map variables are fundamentally Lua values.